tests/testthat/test-fct_match.R

```diff
@@ -0,0 +1,25 @@
+context("test-fct_match.R")
+
+test_that("correct matching as expected", {
+  f <- factor(c("a", "b", "c d"), levels = c("a", "b", "c d", "e"))
+
+  expect_equal(fct_match(f, "a", "b"), c(TRUE, TRUE, FALSE))
+  expect_error(fct_match(f, "a", "x"), regexp = 'not present.*"x"')
```

Comment:
Please don't name the second argument here.

Reply:
Done. I assumed that would be helpful to distinguish it from the tested expression, but no worries if not.